src/super_gradients/training/utils/videos.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Optional, Tuple
+import cv2
+import os
+
+import numpy as np
+
+SUPPORTED_FORMATS = (".avi", ".mp4", ".mov", ".wmv", ".mkv")
+
+
+__all__ = ["load_video", "save_video"]
+
+
+def load_video(file_path: str, max_frames: Optional[int] = None) -> List[np.ndarray]:
+    """Open a video file and extract each frame into numpy array.
```

Comment:
OpenCV loads images as BGR. I believe this should be reflected in documentation if it's intended. Or BGR->RGB conversion should be done inside. Save for image saving. When saving
OpenCV assumes input image is in BGR order.

Reply:
Fixed in d2be71775d822f2a304b9f73044118add2ecaefd